src/common/MovieTile/styled.js

```diff
@@ -0,0 +1,118 @@
+import styled from "styled-components";
+import { ReactComponent as StarIcon } from "../../common/img/star.svg";
```

Comment:
what do you think about unifying the naming and changing folders with image files from "img" to "assets"? :)